libpdbg/hwunit.h

```diff
@@ -373,4 +373,9 @@ struct adc {
 };
 #define target_to_adc(x) container_of(x, struct adc, target)
 
+struct gpio_expander {
```

Comment:
Commit message : please provide the use case for this hardware.  This is also not processor specific right? genral purpose  bmc IO pin you are referring here ?